beacon-chain/p2p/discovery.go

```diff
@@ -31,7 +31,7 @@ type Listener interface {
 func (s *Service) createListener(
 	ipAddr net.IP,
 	privKey *ecdsa.PrivateKey,
-) *discover.UDPv5 {
+) (*discover.UDPv5, error) {
```

Comment:
If you are returning an error here, you need to call a fatal over here 
https://github.com/prysmaticlabs/prysm/blob/master/beacon-chain/p2p/service.go#L184

Reply:
Done, thank you